BE scraper fails

[BE scraper fails](https://github.com/openZH/covid_19/runs/723239120?check_suite_focus=true):

```
Run the scraper...
BE Error: Unknown date/time format: Verspätete Nachmeldung, April
Traceback (most recent call last):
  File "/home/runner/work/covid_19/covid_19/scrapers/parse_scrape_output.py", line 142, in <module>
    new_date = parse_date(v)
  File "/home/runner/work/covid_19/covid_19/scrapers/scrape_dates.py", line 175, in parse_date
    assert False, f"Unknown date/time format: {d}"
AssertionError: Unknown date/time format: Verspätete Nachmeldung, April
```

--- a/scrapers/scrape_be.py
+++ b/scrapers/scrape_be.py
@@ -125,6 +125,8 @@ for t in soup.find_all('table'):
 
         is_first = True
         for row in [r for r in t.find_all('tr') if r.find_all('td')]:
+            if "Nachmeldung" in row.text:
+                continue
             if not is_first:
                 print('-' * 10)
             is_first = False

--- a/scrapers/scrape_matrix.py
+++ b/scrapers/scrape_matrix.py
@@ -36,7 +36,7 @@ matrix = {
     'VD': ['Confirmed cases', 'Deaths', 'Hospitalized', 'ICU'],
     'VS': ['Deaths', 'Hospitalized', 'ICU', 'Vent'],
     'ZG': ['Confirmed cases', 'Deaths', 'Released', 'Hospitalized', 'ICU'],
-    'ZH': ['Confirmed cases', 'Deaths', 'Hospitalized', 'Vent'],
+    'ZH': ['Confirmed cases', 'Deaths', 'Hospitalized'],
     # 'FL': [],  # No scraper.
 }
 

